apps/server/src/modules/common-cartridge/controller/common-cartridge.controller.ts

```diff
@@ -13,4 +14,14 @@ export class CommonCartridgeController {
 	public async exportCourse(@Param() exportCourseParams: ExportCourseParams): Promise<CourseExportBodyResponse> {
 		return this.commonCartridgeUC.exportCourse(exportCourseParams.parentId);
 	}
+
+	@Get('export/lesson/:parentId')

```

Comment:
Rename parameter to `:id`.

Reply:
this is only for test that the Client works and should be deleted after review

Reply:
Ok, I will resolve this conservation when changes are reverted.